services/uploads/src/antivirus.js

```diff
@@ -32,13 +32,26 @@ async function isS3FileTooBig(s3ObjectKey, s3ObjectBucket) {
   return fileSize > constants.MAX_FILE_SIZE;
 }
 
+let downloadDir = constants.DOWNLOAD_DIR;
+
+/**
+ * Returns the /tmp local path from an s3ObjectKey
+ * @param {string} s3ObjectKey Key of the s3 object
+ * @return {string} Path of the s3 object on the local fs
+ *
+ */
+function pathFromObjectKey(s3ObjectKey) {
+  // remove problematic characters from filename
+  let sanitizedFilename = s3ObjectKey.replace(/[^a-zA-Z0-9]/g, "");
+  return `${downloadDir}/${path.basename(sanitizedFilename)}`;
+}
+
```

Comment:
Sanitizes input file name just in case. Removes `.` too but S3 still reads the file with the name and extension concatenated.